### ISSUE TITLE
GUI.pyw's wmlxgettext asks for an output directory, then warns about overwriting a file

### ISSUE THREAD

**Issue by stevecotton** [wesnoth/wesnoth#7726]
### Game and System Information

- **Version:** 1.17.18+dev
- **Downloaded from:** git
- **Build info:** 1be31358dc8e1fb99965c786a7a67d9b9e240e3f (the post-release version bump to `+dev` after 1.17.18)
- **OS:** Linux

### Description of the bug

GUI.pyw doesn't have a clear understanding of what wmlxgettext's output is. #7624 updated it to ask for a directory, but even if that directory is empty, it then prompts for confirmation to overwrite a *file* (but shows the name of the directory).

### Steps to reproduce the behavior

1. Start GUI.pyw
2. Change to the wmlxgettext tool's tab
3. For the (input) "Directory", choose an add-on
4. Create an empty directory (outside the tool)
5. Choose that empty directory as the "Output dir"
6. Run the tool
7. See a prompt "File {directory_name} already exists. Do you want to overwrite it?"

### Expected behavior

Either no warning message, or a bit more logic and saying which files it might overwrite (the exact list isn't known until the tool scans the input files for textdomains).

For this tool, a warning after running it that it *hasn't* overwritten files might be more useful. As in "there's an old wesnoth-SOH.pot file, but we only updated wesnoth-son_of_haldric.pot instead"; but that would be a feature request.

**Comment by knyghtmare** [comment]
I can confirm this. This happens while trying to make the .pot file for my addons

**Comment by MrTitainin** [comment]
I agree that it should not show such warning message. This often gets me to consider alternative ways (like creating extra directories) just to not lose work by accident.
